.github/workflows/ci.yaml

```diff
@@ -636,6 +636,53 @@ jobs:
           name: ecosystem-result
           path: ecosystem-result
 
+  fuzz-redknot:
+    name: "Check for new red-knot panics"
```

Comment:
This is the name that shows up in the CI jobs list -- I think it should mention fuzzing, it's not clear otherwise what "check for new red-knot panics" actually means

Reply:
Maybe this?

```suggestion
    name: "Fuzz for new red-knot panics"
```

Reply:
or

```suggestion
    name: "Check for new red-knot panics using py-fuzzer"
```

Reply:
Either seems fine to me, up to you!